README.md

```diff
@@ -1,39 +1,45 @@
 
 ![](https://libgdx.com/assets/images/index_showcase/game0.png)
-###### An example for a game created with libGDX: [Pathway](https://store.steampowered.com/app/546430/Pathway/) by Robotality. Find more games in our [Showcase](https://libgdx.com/showcase/).
+###### An example game created with libGDX: [Pathway](https://store.steampowered.com/app/546430/Pathway/) by Robotality. Discover more captivating games in our [Showcase](https://libgdx.com/showcase/).
 
-## Getting libGDX / Documentation
-As we use Gradle, there's no need to download libGDX itself  &ndash; this can all be done by your favorite build tool. We also offer a [setup tool](https://libgdx.com/dev/#how-to-get-started-with-libgdx), which automatically creates a project for you and downloads everything necessary. See our **[website](https://libgdx.com/wiki/start/setup)** for instructions on how to get started with this or read our **[wiki](https://libgdx.com/wiki/)**.
+## Getting Started with libGDX / Documentation
+Thanks to Gradle, you can easily set up libGDX without the need to download the framework itself. Your favorite build tool can handle everything for you. Additionally, we offer a convenient [setup tool](https://libgdx.com/dev/#how-to-get-started-with-libgdx) that automates project creation and downloads all the necessary components. Check out our **[website](https://libgdx.com/wiki/start/setup)** for instructions on
+
+ getting started or refer to our comprehensive **[wiki](https://libgdx.com/wiki/)**.
 
 - [Creating a libGDX Project](https://libgdx.com/wiki/start/setup)
-- [A Simple Game](https://libgdx.com/wiki/start/a-simple-game)
+- [Building a Simple Game](https://libgdx.com/wiki/start/a-simple-game)
 - [Tutorials & Demos](https://libgdx.com/wiki/start/demos-and-tutorials)
 
-We also provide [javadocs](https://javadoc.io/doc/com.badlogicgames.gdx) online. The javadocs are bundled with every libGDX distribution for consumption in your favorite IDE.
+We provide the libGDX [javadocs](https://javadoc.io/doc/com.badlogicgames.gdx) online for easy reference. Additionally, the javadocs are bundled with every libGDX distribution, ensuring smooth integration with your preferred IDE.
 
-## Community & Contributing
-You can follow the **latest news** about libGDX on our [blog](https://libgdx.com/news/). Another good way to get in touch with our community is to join the official [libGDX Discord](https://libgdx.com/community/discord/).
+## Community & Contribution
+Stay up to date with the latest libGDX news by following our [blog](https://libgdx.com/news/). For engaging discussions and support, join our official [libGDX Discord](https://libgdx.com/community/discord/).

```

Comment:
I think "latest libGDX news" should be in bold, as it was before.

Reply:
Ok, I will change it :)

Reply:
Hey @crykn @tommyettinger I'm new to open source, can you tell me how to change commits after pull request as I've made or I've to create new branch and commit new pull request?